### PR TITLE
feat(starter-s3): saving String content directly to S3BucketRepository

### PR DESCRIPTION
**Pull request by sdafox**

### CODE REVIEW

File: sda-commons-starter-s3/src/test/java/org/sdase/commons/spring/boot/s3/config/S3BucketRepositoryTest.java [hunk 1]
```diff
@@ -77,11 +78,32 @@ void shouldThrowFileNotFoundException() {
...
+    IOException exception = assertThrows(IOException.class, () -> subject.save(key, null));
+
+    String expectedMessage = String.format("Content to be saved by key %s must not be null!", key);
+    String actualMessage = exception.getMessage();
+
+    assertTrue(actualMessage.contains(expectedMessage));
+  }
+
+  @Test
+  void shouldSaveContentAsUTF8() throws IOException {
```
Comment (JoergSiebahn) [inline sda-commons-starter-s3/src/test/java/org/sdase/commons/spring/boot/s3/config/S3BucketRepositoryTest.java:93]: NIT: How is this test related to UTF8?

Comment (JoergSiebahn) [reply sda-commons-starter-s3/src/test/java/org/sdase/commons/spring/boot/s3/config/S3BucketRepositoryTest.java:93]:
```suggestion
  void shouldSaveContentFromString() throws IOException {
```

**Comment by christopher-cudennec** [comment]
Could you squash your commits and adjust your commit message? I would suggest `feat(starter-s3): ...` as prefix.

I must admit that commit messages in this repo are not as clean as they are in sda-dropwizard-commons. Usually they should use the semantic prefix and the submodule as context.
